src/ui/server/helpers/companies-house-search/validation/rules/companiesHouseNumber.ts

```diff
@@ -16,7 +16,9 @@ const { ELIGIBILITY } = ERROR_MESSAGES.INSURANCE;
  * - only contains letters and/or numbers
  * - has a length greater than 6
  */
-const schema = Joi.string().alphanum().min(6).required();
+const joiFunc = Joi.string();
```

Comment:
not a blocker, but should we have a more descriptive name?

Reply:
Great shout - will fix in next PR